darwin/client.py

```diff
@@ -797,7 +819,7 @@ def post_workflow_comment(
 
         return comment_id
 
-    def instantiate_item(self, item_id: int) -> int:
+    def instantiate_item(self, item_id: int, include_metadata: bool = False) -> Union[int, Tuple[int, DatasetItem]]:
```

Comment:
This looks weird, but I didn't want to make backwards incompatible change here, not sure if this is used in SDK mode. @simedw maybe you can say if that's sensible approach, or maybe that's not part of public part of SDK and I can just refactor this to always return DatasetItem.

Reply:
`instantiate_item` is not really needed in v2 as an item should always be in workflow.

Reply:
This is for V1 only.

Reply:
I mean, this change. This PR addresses both versions.